### PR TITLE
Update type definitions for OneSchemaTemplateOverrides

### PR DESCRIPTION
**Pull request by matthewoey**

### CODE REVIEW

File: packages/importer/src/config.ts
Comment (jennkao) [inline packages/importer/src/config.ts:150]: ah is the `Omit` necessary here since this type also defines label?

Comment (matthewoey) [reply packages/importer/src/config.ts:150]: Michael and I discussed that `Omit` is more idiomatic for typescript since it can help make it more clear that this field is being replace.

**Review by jennkao — APPROVED** [review]
typescript changes look fine, approving to unblock but also unfamiliar about the sdk publishing process - would defer to you or @20michaelz there!

**Pull request merged**
